docs/git-branch-setup.md

```diff
@@ -0,0 +1,86 @@
+<!--
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+-->
+
+## Git branch setup
+
+This document describes how to set up your development environment and contribute changes to
+[accessibility-insights-web](https://github.com/Microsoft/accessibility-insights-web). This document assumes basic working knowledge with Git and related tools.
+We are providing instructions specific to this project.
+
+## Creating your own fork
+
+If you wish to contribute changes back to the [accessibility-insights-web](https://github.com/Microsoft/accessibility-insights-web) repository, start by creating your own fork of the repository. This will keep the number of branches on the main repository to a small count. In your own fork, you can create as many branches as you like.
+
+-   Navigate to [GitHub](https://github.com/) with a browser and log in to your GitHub account. For the sake of this document, let's assume your username is **adacat**.
+-   Navigate to the [accessibility-insights-web](https://github.com/Microsoft/accessibility-insights-web) repository in the same browser session.
+-   Click on the **Fork** button at the top right corner of the page.
+-   Create the fork under your account. Your GitHub profile should now show **accessibility-insights-web** as one of your repositories.
+-   Create a folder on your device and clone your fork of the **accessibility-insights-web** repository. e.g. `https://github.com/adacat/accessibility-insights-web`. Notice how your GitHub username is in the repository location.
+
+```
+> git clone https://github.com/adacat/accessibility-insights-web
+```
+
+## Setting up the upstream repository
+
+Before starting to contribute changes, please setup your upstream repository to the
+primary **accessibility-insights-web** repository.
+
+-   When you run git remote -v, you should see only your fork in the output list
+
+```
+> git remote -v
+
+    origin  https://github.com/adacat/accessibility-insights-web (fetch)
```

Comment:
I think it would make sense to remove the indent and the extra blank lines in the sample output here (I know they exist in the Office Fabric version of the doc, but the actual output doesn't have them).

Reply:
Updated to match the actual format from command line.